src/crawler/scraper.py

```diff
@@ -24,16 +24,96 @@
 from enum import Enum
 
 
+class Count:
+    def __init__(self, name_of_app, package_name,
+                 country, lang):
+        self.name_of_app = name_of_app
+        self.package_name = package_name
+        self.lang = lang
+        self.country = country
+
+    def __get_file_name(self, score):
+        file_name = 'review_' + self.package_name + \
+            '_' + self.country + '_'+self.lang+'_'+str(score) + '.csv'
+        return file_name
+
+    def __get_count_file_name(self, duration):
+        file_name = duration + '_review_count_' + self.package_name + \
+            '_' + self.country + '_'+self.lang+'_' + '.csv'
+        return file_name
+
+    def __should_count(self, duration):
+        file = self.name_of_app + '//' + self.__get_count_file_name(duration)
+        return Utils.is_file_present(file)
+
+    def count_reviews_each_month(self):
+        if self.__should_count('month'):
+            return
+        dir_name = self.name_of_app
+        file_name = self.__get_file_name(1)
+        path = ".\\" + dir_name + "\\" + file_name
```

Comment:
This code may not run in Unix-based
[diff.txt](https://github.com/aicore/app_info_extracter/files/6329656/diff.txt)
 OS. Could you change it

Reply:
Done